src/modules/ui/screen/screen.view.ts

```diff
@@ -88,22 +128,158 @@ class ScreenView extends View<IScreenViewStyles>
   }
 
   hideCursor() {
-    this._$node.classList.add(this.styleNames.hiddenCursor);
+    toggleNodeClass(this._$node, this.styleNames.hiddenCursor, true);
   }
 
   showCursor() {
-    this._$node.classList.remove(this.styleNames.hiddenCursor);
+    toggleNodeClass(this._$node, this.styleNames.hiddenCursor, false);
+  }
+
+  setViewMode(viewMode: ViewMode) {
```

Comment:
as for me `ScreenView` little bit "smarter" then it should be.

Mb move some logic to controller?

`start/stop` for example:
https://github.com/wix/playable/pull/63/files#diff-8b93438085d5271432430d46cdd5f75fR287
https://github.com/wix/playable/pull/63/files#diff-8b93438085d5271432430d46cdd5f75fR155